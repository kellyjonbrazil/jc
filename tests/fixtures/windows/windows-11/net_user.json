{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator"
    },
    {
      "user_name": "user1"
    },
    {
      "user_name": "DefaultAccount"
    },
    {
      "user_name": "Guest"
    },
    {
      "user_name": "WDAGUtilityAccount"
    }
  ],
  "account_origin": "\\\\DESKTOP-WIN11-HOME"
}
