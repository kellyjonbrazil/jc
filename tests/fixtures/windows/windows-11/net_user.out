User accounts for \\DESKTOP-WIN11-HOME

-------------------------------------------------------------------------------
Administrator            user1                    DefaultAccount
Guest                    WDAGUtilityAccount
The command completed successfully.